pages/mesosphere/dcos/services/pxc/0.2.1-5.7.21/release-notes/index.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes
+title: Release Notes
+menuWeight: 10
+excerpt: Discover the new features, updates, and known limitations in this release of the Percona XtraDB Cluster Service
+featureMaturity:
+enterprise: false
+model: /mesosphere/dcos/services/pxc/data.yml
+render: mustache
+---
+
+# Release Notes for Percona XtraDB Cluster Service version 0.2.1-5.7.21
+
```

Comment:
We can add category here like ###Hot Fix or ### Improvements.

Reply:
Done.